### PR TITLE
Display swap txs

### PR DESCRIPTION
**Pull request by nop33**

### CODE REVIEW

File: src/components/Amount.tsx [hunk 1]
```diff
@@ -18,103 +18,106 @@ along with the library. If not, see <http://www.gnu.org/licenses/>.
...
-      const amountParts = amount.split('.')
-      integralPart = amountParts[0]
-      fractionalPart = amountParts[1]
+    if (fadeDecimals && ['K', 'M', 'B', 'T'].some((char) => amount.endsWith(char))) {
+      quantitySymbol = amount.slice(-1)
+      amount = amount.slice(0, -1)
     }
-  } else {
-    return (
-      <span className={className} tabIndex={tabIndex ?? -1} data-tip={value?.toString()}>
```
Comment (nop33) [inline src/components/Amount.tsx:71]: just realized I forgot to port the tooltips. On it.

Comment (nop33) [reply src/components/Amount.tsx:71]: fixed

File: src/components/Amount.tsx [hunk 2]
```diff
@@ -18,103 +18,114 @@ along with the library. If not, see <http://www.gnu.org/licenses/>.
...
 import styled from 'styled-components'
+
+import { convertToPositive } from '@/utils/numbers'
+
 interface AmountProps {
   value?: bigint | number
   decimals?: number
   isFiat?: boolean
-  unknownToken?: boolean
+  isUnknownToken?: boolean
```
Comment (mvaivre) [inline src/components/Amount.tsx:28]: 👍

File: src/types/assets.ts [hunk 3]
```diff
@@ -16,17 +16,38 @@ You should have received a copy of the GNU Lesser General Public License
...
 */
 
+import { TransactionDirection, TransactionInfoType } from '@alephium/sdk'
 import { AddressBalance, Token } from '@alephium/sdk/api/explorer'
 import { TokenInfo } from '@alephium/token-list'
+import { Output } from '@alephium/web3/dist/src/api/api-explorer'
 
+import { PartialBy } from '@/types/generics'
+
+// TODO: Add in SDK?
```
Comment (mvaivre) [inline src/types/assets.ts:26]: yes, definitely imo

File: src/types/generics.ts [hunk 4]
```diff
@@ -18,4 +18,5 @@ along with the library. If not, see <http://www.gnu.org/licenses/>.
 
 export type OnOff = 'on' | 'off'
 
+// TODO: Add in SDK?
```
Comment (mvaivre) [inline src/types/generics.ts:21]: Mhh this I'm not sure. It has nothing to do with Alephium.

**Review by mvaivre — COMMENTED** [review]
Code looks good, will do some functional testing now!

File: src/components/AssetLogo.tsx [hunk 5]
```diff
@@ -32,12 +32,12 @@ interface AssetLogoProps {
...
     {asset.logoURI ? (
-      <LogoImage src={asset.logoURI ?? AlephiumLogoSVG} />
+      <LogoImage src={asset.logoURI} />
     ) : asset.id === ALPH.id ? (
       <LogoImage src={AlephiumLogoSVG} />
-    ) : asset.id ? (
-      <AssetLogoPlaceholder size={size}>?</AssetLogoPlaceholder>
-    ) : null}
+    ) : (
+      <Ghost size={size * 0.7} />
```
Comment (mvaivre) [inline src/components/AssetLogo.tsx:39]: Let's go for 0.6 if you don't mind. I find 0.7 a bit too much (details).

**Comment by mvaivre** [comment]
image

Our logo became really smol compared to the version in prod!

File: src/components/AssetLogo.tsx (hunk 5 above)
Comment (mvaivre) [reply src/components/AssetLogo.tsx]: Changed in [4b40581](https://github.com/alephium/explorer/pull/191/commits/4b40581417c185fac62ac725e631a754a6c94108)

**Comment by nop33** [comment]
>  Our logo became really smol compared to the version in prod!

Thanks for spotting this! Fixed!

**Review by mvaivre — APPROVED** [review]
👏

**Pull request merged**